### PR TITLE
Chapter Committee Regional Chapter Policy Update

### PR DESCRIPTION
**Pull request by vanderaj**
Merged the Chapter Committee's regional chapter policy. This doesn't take into account association management platforms, so will need a further change to make it go. 

Minor nips and tucks to the existing policy fixing the issues with student chapter breaks and activity. Additionally, removed old staff titles, and spell / grammar checked (part way done).

### CODE REVIEW

**Review by latf6711 — COMMENTED** [review]
line 26 - The OWASP Foundation is not obliged to reimburse for alternate  (meeting scheduling services)  expenses.
The OWASP Foundation is not obligated to reimburse for the use of alternative platform service expenses.

**Review by latf6711 — CHANGES_REQUESTED** [review]
The OWASP Foundation is not obliged to reimburse for alternate  (meeting scheduling services)  expenses.
The OWASP Foundation is not obligated to reimburse for the use of alternative platform service expenses.

 A list of the current leaders and email addresses must be listed on the Chapter's web page on owasp.org. Leaders are strongly encouraged to use their @owasp.org email address on their chapter pages.
The statement is not in line with the Communication section 
Chapter repository creation is automatically set up with leader name & owasp.org accounts.
Automation to validate leader/membership for owasp.org accounts is tied to the leader md file
Leaders can forward owasp.org email to another platform.

Chapter meetings (traditional meetups)
Chapter meetings - Standard, traditional Virtual, or in-person gatherings or meetups
 
Chapter leaders must post chapter activity information (date, time, and location) on the owasp.org chapter page before the event start date.
(date, time, and location/URL) physical or URL for virtual (zoom)

Regional Leadership
I did not see the Minimum & Maximum number of leaders?

After the new chapter is approved, the chapter leader must:
 Provide GitHub usernames in order to get admin access to your chapter repository.
The incorrect statement is listed in the policy. GitHub ID needs to be included in the ticket when applying for the OWASP chapter.  All leaders are to create and have a GitHub ID and access to the chapter repository for backup. A leader with admin access can grant other leaders with admin access. 

Any changes in chapter leadership should be done by submitting a ticket with all information to the [New Chapter/Leader Request](https://owasporg.atlassian.net/servicedesk/customer/portal/8/group/20/create/105) by one of the existing leaders. If the local chapter leadership is inactive, local chapter members can request new leadership using this form.
A ticket needs to be submitted to complete a change in chapter leadership
request new leadership using this ticket

If a leader needs to step down, they should submit a ticket to the [New Chapter/Leader Request]
To complete any changes in leadership a ticket needs to be submitted
There are currently two types of chapters you can start: city chapters and student chapters. Prospective chapter leaders should familiarize themselves with this policy and the draft [Chapter Handbook](https://owasp.org/www-committee-chapter/#div-resources_for_chapters) prior to submitting the form.
prior to submitting the ticket

File: operational/chapters.md [hunk 1]
```diff
@@ -37,164 +21,318 @@ Chapter leaders serve as the main point of contact for their chapters and are re
...
 
 City chapters are the primary form of the OWASP chapter, with hundreds of chapters worldwide.
 
-- Approved city chapters are named "OWASP «city name»". City chapter names must not be a regional or country name unless the city name is the country name (e.g., Monaco).
-- Chapters shall be defined for a single city only; a new chapter may be denied approval if there is another chapter within 80 km (50 miles).
-- Chapter leaders must reside within 80 km (50 miles) of the chapter location.
-- Exceptions to distance rules may be approved on a case-by-case basis, for example where travel times between two geographically close chapters is excessive (defined as more than one hour).
+- City chapters should be named "OWASP «city name»". City chapter names must not be a regional or country name unless the city name is the country name (e.g., Monaco).
+- City chapters are defined for a single city only.
+- City chapter leaders must reside within 80 km (50 miles) of the chapter location.
```
[the remark this answers is not in the document]
Comment (avidouglen) [reply operational/chapters.md:126]: Change this to *should

[the remark this answers is not in the document]
Comment (aramrami) [reply operational/chapters.md:126]: What about small country?
In Morocco, for example the there only two cities (Rabat and Casablanca) where we can find more than 90% of the company and users interested by cyber security and OWASP activities. For my opinion on Chapter for all the country is enough. Hosting an event in Rabat or Casablanca will cover all attendees and companies.

[the remark this answers is not in the document]
Comment (avidouglen) [reply operational/chapters.md:126]: @aramrami so would a Regional Chapter not be a good fit for Morocco? It seems this is a perfect example of the situation we are trying to solve?

[the remark this answers is not in the document]
Comment (aramrami) [reply operational/chapters.md:126]: Yes I agree. For me regional chapter and city chapter are not good to be fit for Morocco and similar countries.

**Pull request merged**